### PR TITLE
Migration Audit

### PR DESCRIPTION
**Pull request by alexfornuto**
Work Tracked via [Project](https://github.com/pantheon-systems/documentation/projects/8) (sort of)

## Effect
PR includes the following changes:
- Changes as required to all migration content
- Expect to see guides condensed, split up, consolidated, and updated throughout.

## Redirects Required
The following list should be updated upon consolidating one doc page into another:
- [ ] /docs/clone-site => /docs/migrate
- [ ] /docs/migrate-from-wpengine => /docs/migrate
- [ ] /docs/migrate-wordpress => /docs/migrate
- [ ] /docs/migrate-troubleshooting => /docs/migrate
- [ ] /docs/migrate-preserve-history => /docs/migrate-manual
- [ ] /docs/drupal-export => /docs/migrate-manual
- [ ] /docs/wordpress-export => /docs/migrate-manual

### CODE REVIEW

File: source/_docs/migrate.md [hunk 1]
```diff
@@ -4,77 +4,324 @@ description: General instructions for preparing and migrating remotely-hosted Dr
...
 
-* **Review and archive your legacy site's logs.** The root causes for errors or slow transactions occurring on your legacy site will probably be migrated over, so it would be a good time to resolve them prior to the migration. Slow MySQL queries should be targeted for optimization, as a distributed infrastructure adds a small amount of latency on each transaction.
+  </code></pre>
+  </div>
+</div>
 
-* **Review your configuration files** (my.conf, php.ini, .htaccess, etc). Any custom settings will need to be reviewed, as Pantheon provides platform-wide configuration. If, for example, the site allows very long timeouts, hides PHP errors, or changes how mySQL transactions are handled; these changes may not exist after migration, and may lead to unexpected results. Also, NGINX doesn't process the Apache-specific .htaccess file, so any redirects or custom logic within those files will need to be reconsidered.
 
-## Troubleshooting
+### Could not import database, unable to locate a database dump. ###
```
Comment (alexfornuto) [inline source/_docs/migrate.md]: This is one of three sections in `## Troubleshooting` I think we should expand, as they seem to only restate the error message.

File: source/_docs/migrate.md [hunk 2]
```diff
@@ -4,77 +4,324 @@ description: General instructions for preparing and migrating remotely-hosted Dr
...
+**Cause:** The migration tool found more than one potential location for files within the archive. This error also occurs if Drupal's private files directory is not placed within the public directory (`sites/default/files/private`).
+
+**Solution:** All files must be moved into the standard location for your site's CMS (`/sites/default/files` for Drupal, and `/wp-content/uploads` for WordPress). For more details, see [Non-Standard Files Locations](/docs/non-standard-file-paths).
+
+### Multiple site directories found within the import archive. ###
+**Cause:** The migration tool found a multisite installation, which is not supported on the platform.
+
+**Solution:** Refer to [Extracting Sites from a Drupal Multisite](/docs/unwind-multisite/).
+
+### Multiple database dumps found within the import archive. ###
```
Comment (alexfornuto) [inline source/_docs/migrate.md]: This is the second section referenced on Line 192

File: source/_docs/migrate.md [hunk 3]
```diff
@@ -4,77 +4,324 @@ description: General instructions for preparing and migrating remotely-hosted Dr
...
+**Cause:** The migration tool detected multiple MySQL database dumps within the archive.
+
+**Solution:** Ensure that a single MySQL dump is included within the archive.
+
+### Multiple code roots found within the import archive. ###
```
Comment (alexfornuto) [inline source/_docs/migrate.md]: And this is the third.

File: source/_docs/migrate.md [hunk 4]
```diff
@@ -4,77 +4,324 @@ description: General instructions for preparing and migrating remotely-hosted Dr
...
+
+```bash
+Connection to appserver.<ENV>.<Site UUID>.drush.in closed by remote host.
+```
+
+If your database and code compressed are less than 256MB you can exclude the files directory from export using the `--tar-options="--exclude=code/sites/default/files"` flag. Then you can download a backup of the files from the existing site and import the archive to the new site within **<span class="glyphicons glyphicons-server"></span> Database / Files** > **Import**.
 
-#### How do I migrate a local site to Pantheon?
-You can import a WordPress or Drupal site archive via URL (within file size limits) using [Terminus](/docs/terminus):
+### How do I migrate a local site to Pantheon?
```
Comment (alexfornuto) [inline source/_docs/migrate.md]: This section seems to only provide WP-specific answers.

File: source/_docs/migrate.md [hunk 5]
```diff
@@ -4,77 +4,324 @@ description: General instructions for preparing and migrating remotely-hosted Dr
...
+### How long does the WordPress migration process take?
+Most migrations are completed within two hours. The migration time depends solely on the size of your site, so be aware that it may take more or less time than estimated. We will send you an email once your migration is complete. If there are any issues with the migration, we will notify you by email.
+
+### Why doesn't my site archive import correctly from Dropbox?
+Change the end of Dropbox URLs from `dl=0` to `dl=1` so we can import your site archive properly. See the Dropbox documentation: [How do I force a file or folder to download or render on dropbox.com?](https://www.dropbox.com/en/help/201)
+
+### How do I use the Pantheon Migrations plugin with a custom WordPress upstream?
```
Comment (alexfornuto) [inline source/_docs/migrate.md]: I think this section could be clarified. It says that the process is the same, but by clicking on **Create New Site** the user won't get the same options. If I understand correctly, they need to do something else to manually initiate the migration, yes? Does this section need a reference to the Terminus command used in the next section?

File: source/_docs/migrate.md [hunk 6]
```diff
@@ -4,77 +4,324 @@ description: General instructions for preparing and migrating remotely-hosted Dr
...
+If you'd like your existing WordPress site to get one-click updates from your [Custom Upstream](/docs/custom-upstream/), then the migration process will be slightly different. The general process will be the same as a vanilla WordPress site, but start with **Create New Site** instead of **Migrate existing site**. You'll need to manually generate a [machine token](/docs/machine-tokens/) to use with the [Pantheon Migrations plugin](https://wordpress.org/plugins/bv-pantheon-migration/).
+
+### How should I migrate a site with a custom Drupal-based upstream?
+If you'd like your existing Drupal site to get one-click updates from your [Custom Upstream](/docs/custom-upstream/), then the migration process will be slightly different. The general process will be the same as a vanilla Drupal site, but start with **Create New Site** instead of **Migrate existing site**. Then use `terminus site:import <site> <url>` to import your site archive, or follow the [Manual migration](/docs/migrate-manual) instructions if your site archive exceeds file size limits.
+
+### What if I can't use drush on my existing Drupal site?
```
Comment (alexfornuto) [inline source/_docs/migrate.md]: Reminder for @alexfornuto - replace this reference during consolidation.

Comment (alexfornuto) [reply source/_docs/migrate.md]: Done

**Comment by rachelwhitton** [comment]
+1 🚢

**Pull request merged**